app/models/practice.rb

```diff
@@ -136,7 +136,14 @@ def origin_picture_s3_presigned_url(style = nil)
   accepts_nested_attributes_for :video_files, allow_destroy: true, reject_if: proc { |attributes| attributes['url'].blank? || attributes['description'].blank? }
   accepts_nested_attributes_for :difficulties, allow_destroy: true
   accepts_nested_attributes_for :risk_mitigations, allow_destroy: true
-  accepts_nested_attributes_for :timelines, allow_destroy: true, reject_if: proc { |attributes| attributes['timeline'].blank? || attributes['milestone'].blank? }
+  accepts_nested_attributes_for :timelines, allow_destroy: true, reject_if: proc { |attributes|
+    reject = attributes['timeline'].blank?
+    ma_reject = false
+    attributes['milestones_attributes'].each do |i, ma|
```

Comment:
we throw a `500 Internal Server Error` here if we try to update one timeline with a milestone but leave the others blank...
- Do we want to support saving the practice ONLY when there is a milestone under each timeline item? If so, I think we can do better error handling instead of throwing a 500

Reply:
Can you post a screen recording of this? So if you try to update an existing timeline entry's milestone, but leave which others blank? Yes, we only want to save the timeline entry if it has a timeline and at least one milestone. Sorry, I'm not getting a 500 error, so I not sure how to respond.

Reply:
will do!

Reply:
sent you a recording via DM

Reply:
Should be fixed. Let me know if it's working as intended now :)

Reply:
thank you!!